doc/dataselect.md

```diff
@@ -133,6 +133,8 @@ dataselect [options] file1 [file2 file3 ...]
 
 <b>-BUD </b><i>directory</i>
 
+<b>-SDS </b><i>directory</i>
```

Comment:
This file is generated from the dataselect.1 man page, so please make the changes to the man page and I'll update the markdown version.